centos_playground/onboard/onboard.py

```diff
@@ -26,7 +26,9 @@ def run(self):
         print(self.converter.result)
         with open('/in/result.yml', 'a+') as out:
             out.write(f'{self.converter.result}\n')
-        if self.converter.result.get('error') or self.converter.result.get('conditional_patch'):
+        if not self.converter.result or \
+                self.converter.result.get('error') or \
+                self.converter.result.get('conditional_patch'):
```

Comment:
I am maybe thinking to much about explicitenes..;)

```suggestion
                "error" in self.converter.result or \
                "conditional_patch" in self.converter.result:
```

And btw, `black` can help you to get rid of those `\` at the end of each line...